src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String STRING_BUY = "buy";
+    private static final String STRING_SUPPLY = "supply";
+    private static final String STRING_СOMA = ",";
+    private static final int ZERO = 0;
+    private static final int OPERETION_INDEX = 0;
+    private static final int ONE = 1;
+    private static final int AMOUNT_INDEX = 1;
+    private static final int TWO = 2;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File outputFile = new File(fromFileName);
+        int totalBuy = 0;
+        int totalSupply = 0;
+        totalSupply = getSupplyAndBuyFromFile(outputFile)[0];
+        totalBuy = getSupplyAndBuyFromFile(outputFile)[1];
+        File inputFile = new File(toFileName);
+        writeToFile(totalSupply, totalBuy, inputFile);
+    }
+
+    private static int[] getSupplyAndBuyFromFile(File fileReader) {
+        try {
+            List<String> lines = Files.readAllLines(fileReader.toPath());
+            return addSupplyAndBuyto(lines);
+
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file", e);
+        }
+    }
```

Comment:
```suggestion
    private static List<String> readFromFile(File inputFile) {
        try {
            return Files.readAllLines(inputFile.toPath());
        } catch (IOException e) {
            throw new RuntimeException("Can't read data from the file: " + inputFile.getName(), e);
        }
    }
```
change namings to more logical ones
don't call counting method here, this method (`readFromFile()`) should only read from file
more informative messages in `throw` clause